src/Beberlei/Metrics/Collector/Graphite.php

```diff
@@ -101,7 +103,12 @@ public function flush()
         $this->data = array();
     }
 
-    public function push($stat, $value, $time = null)
+    /**
+     * @param string $stat
+     * @param string $value
+     * @param int|null $time
+     */
+    private function pushStat($stat, $value, $time = null)
```

Comment:
If you really want to remove push from the public API, juste add a `@deprected` on it.

Anyway, I **want** do keep this method in the public API. I added the graphite collector in this project to be able to "replay" old data.

More over, Graphite handler is not meant to be used in production (better use statsd). So basically, the only way I use this collector is through the `push` method.